x-pack/plugins/spaces/server/lib/space_request_interceptors.test.js

```diff
@@ -6,18 +6,21 @@
 import sinon from 'sinon';
 import { Server } from 'hapi';
 import { initSpacesRequestInterceptors } from './space_request_interceptors';
-import { createSpacesService } from './create_spaces_service';
 
 describe('interceptors', () => {
   const sandbox = sinon.sandbox.create();
   const teardowns = [];
+  const headers = {
```

Comment:
curious: why are auth headers needed here?

edit: does this enable the tests below which verify we are passing along the client credentials to the underlying SOC/repository?

Reply:
Yup, it's just to verify that we're passing the `request` to the methods that need it. HapiJS doesn't pass the injected `request` itself through, but is using the values from that to create their own request object, so this lets us verify the actual parts that we need passed through are.